src/tox/config/__init__.py

```diff
@@ -410,6 +408,19 @@ def __setitem__(self, name, value):
         self.definitions[name] = value
         self.resolved[name] = value
 
+    def items(self):
+        return [(name, self.__getitem__(name)) for name in self.definitions]
+
+    def export(self):
+        # post-process items to avoid internal syntax/semantics
+        # such as {} being escaped using \{\}, suitable for use with
+        # os.environ .
+        return {
+            name: re.sub(r"\\({|})", r"\1", value)
```

Comment:
regex is overkill here, I'd just use `.replace` -- it looks like the original code did this why did you change it? (the original code is much more readable imo)

Reply:
I didnt pay attention to the original code, as it was in a very different area of the code exercised by different tests.  I'm happy to use `.replace`.

Reply:
Yeah please do.

Reply:
This was done.  Please review again when you have time, as this PR merge conflicts with other PRs, and is also a prerequisite for testing in other PRs using brackets in settings and their effect on the runtime environment variables present while running `commands`.